coderd/database/queries/auditlogs.sql

```diff
@@ -3,12 +3,21 @@
 -- name: GetAuditLogsOffset :many
 SELECT
     audit_logs.*,
+    -- sqlc.embed(users) would be nice but it does not seem to play well with
+    -- left joins.
```

Comment:
Yea, there is a few issues open on this:

https://github.com/sqlc-dev/sqlc/issues?q=is%3Aissue+is%3Aopen+nullable+embed

:cry: